server/src/main/java/org/elasticsearch/action/ActionModule.java

```diff
@@ -450,10 +447,8 @@ public ActionModule(Settings settings, IndexNameExpressionResolver indexNameExpr
                 restWrapper = newRestWrapper;
                 if (restWrapper.getClass().getCanonicalName() == null ||
                     restWrapper.getClass().getCanonicalName().startsWith("org.elasticsearch") == false) {
-                    deprecationLogger.deprecate(DeprecationCategory.PLUGINS, "3rd_party_rest_deprecation", "The " +
-                        plugin.getClass().getName() + "plugin installs a custom REST wrapper. This functionality is deprecated and will " +
-                        "not be possible in Elasticsearch 8.0. If this plugin is intended to provide security features for Elasticsearch " +
-                        "then you should switch to using the built-in Elasticsearch features instead.");
+                    logger.warn("The " + plugin.getClass().getName() + "plugin installs a custom REST wrapper. This functionality " +
+                        "should be disabled");
```

Comment:
I don't think this wording is right - what does _should be disabled_ mean to the reader?

Take a step back - Who is going to read this message, and what action do you want them to take when they do?

The warning message should convey that information to them, as succinctly as it can.